sdk/src/orderSubscriber/OrderSubscriber.ts

```diff
@@ -106,7 +110,13 @@ export class OrderSubscriber {
 					'User',
 					buffer
 				) as UserAccount;
-
+			this.eventEmitter.emit(
+				'onUpdate',
+				userAccount,
+				userAccount.orders.filter((order) => order.slot.toNumber() === slot),
```

Comment:
this could be empty if a user just had partial fill

Reply:
oh gotcha. okay i'll check and make sure that the array isn't empty before emitting an event